src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User as UserItem } from '../../types/User';
```

Comment:
_comment_ 
same as before. Renaming imported types makes a lot of mess and decreases code clarity a lot.
```suggestion
import { User } from '../../types/User';
```

Reply:
thank you!